gradoop-common/src/test/java/org/gradoop/common/GradoopTestUtils.java

```diff
@@ -165,6 +171,29 @@
     SUPPORTED_PROPERTIES.put(KEY_f, SET_VAL_f);
   }
 
+  public static ElementFactoryProvider<EPGMGraphHead, EPGMVertex, EPGMEdge> getElementFactoryProvider() {
```

Comment:
JavaDoc missing. 
Also rename to `getEPGMElementFactoryProvider()`
Also create a static attribute to avoid multiple creations of the provider object. On each call of this mehtod, check if it is initialized ... if not - create.